packages/schematics/schematics/blank/project-files/__dot__npmignore

```diff
@@ -0,0 +1,3 @@
+# Ignores TypeScript files, but keeps definitions.
+**/*.ts
+!**/*.d.ts
```

Comment:
`**/` isn't needed.  It should be recursive unless prefixed with a `/`.

Reply:
Done.